reqwest-middleware/src/client.rs

```diff
@@ -527,6 +527,7 @@ impl RequestBuilder {
     /// The [request mode][mdn] will be set to 'no-cors'.
     ///
     /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/API/Request/mode
+    #[cfg(target_family = "wasm")]
```

Comment:
reqwest uses this cfg attribute https://github.com/seanmonstar/reqwest/blob/master/src/lib.rs#L255C9-L255C39:
```
#[cfg(target_arch = "wasm32")]
```

I wonder if the issue is that these don't match